deploy/utils.sh

```diff
@@ -157,6 +155,147 @@ function deploy_env {
+}
+
+fetch_ssl_from_conjur() {
+  selector="role=follower"
+  cert_location="/opt/conjur/etc/ssl/conjur.pem"
+  if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+    selector="app=conjur-cli"
+    export cert_location="/root/conjur-${CONJUR_ACCOUNT}.pem"
+  fi
+
+  export conjur_pod_name=$($cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
+}
+
+setup_helm_environment() {
+  set_namespace $CONJUR_NAMESPACE_NAME
+
+  configure_conjur_url
+
+  fetch_ssl_from_conjur
+
+  ssl_location="conjur.pem"
+  if [ "${DEV}" = "true" ]; then
+    ssl_location="../conjur.pem"
+  fi
+  # Save cert for later setting in Helm
+  $cli_with_timeout "exec ${conjur_pod_name} --namespace $CONJUR_NAMESPACE_NAME cat $cert_location" > "$ssl_location"
+
+  set_namespace $APP_NAMESPACE_NAME
+}
+
+set_image_path() {
+  image_path="${APP_NAMESPACE_NAME}"
+  if [[ "${PLATFORM}" = "openshift" && "${DEV}" = "false" ]]; then
+    # Image path needs to point to internal registry path to access image
+    image_path="docker-registry.default.svc:5000/${APP_NAMESPACE_NAME}"
+  elif [[ "${PLATFORM}" = "kubernetes" && "${DEV}" = "false" ]]; then
+    image_path="${DOCKER_REGISTRY_PATH}/${APP_NAMESPACE_NAME}"
+  fi
+  export image_path
+}
+
+fill_helm_chart() {
+  helm_path="."
+  if [ "${DEV}" = "false" ]; then
+    helm_path=".."
+  fi
+
+  set_image_path
+
+  id=${1:-""}
+  sed -e "s#{{ SECRETS_PROVIDER_ROLE }}#${SECRETS_PROVIDER_ROLE:-"secrets-provider-role"}#g" \
+    -e "s#{{ SECRETS_PROVIDER_ROLE_BINDING }}#${SECRETS_PROVIDER_ROLE_BINDING:-"secrets-provider-role-binding"}#g" \
+    -e "s#{{ CREATE_SERVICE_ACCOUNT }}#${CREATE_SERVICE_ACCOUNT:-"true"}#g" \
+    -e "s#{{ SERVICE_ACCOUNT }}#${SERVICE_ACCOUNT:-"secrets-provider-service-account"}#g" \
+    -e "s#{{ K8S_SECRETS }}#${K8S_SECRETS:-"test-k8s-secret"}#g" \
+    -e "s#{{ CONJUR_ACCOUNT }}#${CONJUR_ACCOUNT:-"cucumber"}#g" \
+    -e "s#{{ CONJUR_APPLIANCE_URL }}#${CONJUR_APPLIANCE_URL:-"https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api"}#g" \
+    -e "s#{{ CONJUR_AUTHN_URL }}#${CONJUR_AUTHN_URL:-"https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api/authn-k8s/${AUTHENTICATOR_ID}"}#g" \
+    -e "s#{{ CONJUR_AUTHN_LOGIN }}# ${CONJUR_AUTHN_LOGIN:-"host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${APP_NAMESPACE_NAME}/*/*"}#g"  \
+    -e "s#{{ SECRETS_PROVIDER_SSL_CONFIG_MAP }}# ${SECRETS_PROVIDER_SSL_CONFIG_MAP:-"secrets-provider-ssl-config-map"}#g" \
+    -e "s#{{ IMAGE_PULL_POLICY }}# ${IMAGE_PULL_POLICY:-"IfNotPresent"}#g" \
+    -e "s#{{ IMAGE }}# ${IMAGE:-"$image_path/secrets-provider"}#g" \
+    -e "s#{{ TAG }}# ${TAG:-"latest"}#g" \
+    -e "s#{{ LABELS }}# ${LABELS:-"app: test-helm"}#g" \
+    -e "s#{{ DEBUG }}# ${DEBUG:-"false"}#g" \
+    -e "s#{{ RETRY_COUNT_LIMIT }}# ${RETRY_COUNT_LIMIT:-"3"}#g" \
+    -e "s#{{ RETRY_INTERVAL_SEC }}# ${RETRY_INTERVAL_SEC:-"30"}#g" \
```

Comment:
I can't look at it...
I suggested to make use of `envsubst`.
Have you tried it?